src/main/java/com/watch/shop/app/view/WatchView.java

```diff
@@ -0,0 +1,88 @@
+package com.watch.shop.app.view;
+
+import com.watch.shop.app.model.Color;
+import com.watch.shop.app.model.Watch;
+import com.watch.shop.app.model.WatchType;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Scanner;
+
+public class WatchView {
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public String getUserChoice() {
+        String menu = """
+                
+                Choose an action:
+                1. Display all watches
+                2. Sort by parameter and display all watches
+                3. Display total cost of watches
+                4. Add a new watch
+                5. Exit
+                """;
+        System.out.println(menu);
+        return scanner.nextLine();
+    }
+
+    public String sortByParam() {
+        String menu = """
+                
+                Choose a parameter:
+                1. Price
+                2. Color
+                3. Arrival Date
+                4. Exit
+                """;
+        System.out.println(menu);
+        return scanner.nextLine();
+    }
+
+    public void displayAllWatches(List<Watch> watches) {
+        System.out.println("_".repeat(97));
```

Comment:
I believe it could be extracted into constants.

Please make separate file Constants.java, put it into View package, and use it as container for all String constants as public static final String MAIN_MENU = ... ;
 SEPARATOR = "_".repeat(97).
 
 In this case, we will have all constants in one place and we could reuse them without duplicating, and for example when we need to add Internaliztion it's easier to translate.
 
 As you see across comments, Java is used widely for building large application, so everything is around of structured and group - isolate functionality  for reusability. 
 
 In small application it could be appeared, that it's fine, you can still manage even messed code with crossed functionality.
  But just scale your application in x100 or x1000 size, and you will see that this approach will not work.
  Developers 50% of his time read code, 30% seeking solution, read  documentation, 10-30% meetings, and only 10% writes code.
  
  We write code for others, and others write code for us, so we always will be working with unknown code, and it will be much easier if it will be structured, well grouped, divided by responsibilities, layers and so on, so we just could quickly find the right place and just reuse component.